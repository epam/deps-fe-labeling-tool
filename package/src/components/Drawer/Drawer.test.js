import React from 'react'
import { shallow } from 'enzyme'
import { Drawer } from '.'

describe('Component: Drawer', () => {
  const wrapper = shallow(
    <Drawer>
      <div>children</div>
    </Drawer>
  )

  it('should render correct layout', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
