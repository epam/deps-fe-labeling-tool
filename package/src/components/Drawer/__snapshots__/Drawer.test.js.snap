// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Drawer should render correct layout 1`] = `
<DrawerWrapper
  getContainer={false}
>
  <div>
    children
  </div>
</DrawerWrapper>
`;
